.github/workflows/push-to-npm-gpr.yml

```diff
@@ -28,3 +28,15 @@ jobs:
         run: npm publish --access public mitre-saf-v*.tgz
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      # Setup .npmrc file to publish to GitHub Package Registry
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@mitre'
```

Comment:
can you gimme a documentation link for what scope does?  couldn't quickly find it on the github repo for this action.

in any case, why do we have it here and not on the one for the npm package as well?

Reply:
https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages

Reply:
how does our current publishing to npm work without the scope variable?  is the scope variable even required?  does it do more under the hood than just prepend the '@mitre' to the package name?

Reply:
My take is that the publish to `npm` does not need the scope, if it is working don't mess with it...lol

Reply:
https://docs.npmjs.com/cli/v9/using-npm/scope#associating-a-scope-with-a-registry

After reading a whole bunch and checking out code, it seems like the scope stuff is multipart.  The first part is that a repo is automatically "scoped" by having "@whatever/" as part of the package name in the package.json, which we do with the "@mitre/".  Second, for npmjs after you publish a package, you can add it to your 'team' there so that it is officially part of your organization.  Third, for github that's not as relevant since these packages are associated with the repos and accounts they're created from.  Fourth, you can locally associate scopes to registries such that when you publish a package it is automatically sent to that particular registry.  However, since we re-initialize that npmrc file every time and point it to a specific registry via the action, I do not think we need to do that local association.

In conclusion, we can safely remove the 'scope' input and it should work.  If it turns out to not do what we want on our next release, we'll know how to fix it, and we could manually push up the release.